test/config.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
No.

The test scripts all live in the same directory as `config.sh`, so why not just use `$0`?